@here/harp-map-controls/lib/Utils.ts

```diff
@@ -86,3 +86,35 @@ export function getWidthAndHeightFromCanvas(
         height: safeParseDecimalInt(canvas.style.height, canvas.clientHeight)
     };
 }
+
+export interface CameraPoseOptions {
```

Comment:
You can use a partial type, instead of having to re type the properties, see: https://www.typescriptlang.org/docs/handbook/utility-types.html#partialt

Reply:
done